src/ticks/ticks.h

```diff
@@ -13,6 +13,9 @@
 #include "uthash.h"
 #include "utlist.h"
 
+#define SYM_TAB_SIZE    0x10000
+#define BUFF_SIZE       0x10000
```

Comment:
BUFF_SIZE might as well live in disassembler_alg.

Reply:
Although it is also used in disassembler_main.c to allocate the buffer.